receiving_collection.json

```diff
@@ -1,44 +1,45 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "description": "collection of receivings",
+  "description": "A collection of receivings",
   "type": "object",
   "properties": {
-    "receivings": {
-      "description": "an array of receivings",
-      "id": "receivings",
+    "received_items": {
+      "description": "List of receivings",
+      "id": "received_items",
       "type": "array",
       "items": {
         "type": "object",
         "properties": {
           "po_line_id": {
-            "description": "UUID of po line this receiving is related to",
+            "description": "The id of the receiving po line",
             "type": "string",
             "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
           },
           "received": {
-            "description": "number of items received",
+            "description": "The number of items received",
             "type": "integer"
           },
           "items": {
-            "description": "an array of items details",
+            "description": "List of recieving items details",
+            "id": "items",
             "type": "array",
-            "items": {
+            "item": {
```

Comment:
Please change `item` to `items` because jaxrs-code-generator will not be able to create proper model but `Object` will be used instead i.e. the result is
```java
    /**
     * List of recieving items details
     * 
     */
    @JsonProperty("items")
    @JsonPropertyDescription("List of recieving items details")
    @Valid
    private List<Object> items = new ArrayList<Object>();
```